examples/tf_track_distribution.py

```diff
@@ -0,0 +1,100 @@
+import aim
+from aim import track
+
+import tensorflow as tf
+
+# Import MNIST data
+from tensorflow.examples.tutorials.mnist import input_data
```

Comment:
I've got problems with this line. See [here](https://github.com/tensorflow/tensorflow/issues/28801)